lang/french.json

```diff
@@ -294,6 +469,7 @@
     "sydneyhud_save_preplanning_desc" : "Sauvegarde votre planification",
     "sydneyhud_show_assault_states" : "Montrer les États d'assaut",
     "sydneyhud_show_assault_states_desc" : "Montre les États d'assaut (Contrôle, Anticipation, Développement, Support et Retrait) dans la boîte d'assaut\nCette option n'est pas disponible en cas d'assaut sans fin et quand le capitaine est présent",
+      "sydneyhud_show_assault_states_disabled_desc" : "Le menu est désactivé et les options dans le menu ne peuvent être changés pendant que BAI est activé\nChangez ces options dans le menu de BAI",
```

Comment:
This string was replaced with "sydneyhud_hud_tweaks_assault_disabled_desc" and also the string description was changed.

Reply:
So, I suppose I need to remove it ?

Reply:
More like replace that string with this: